influxdb3_server/src/http.rs

```diff
@@ -369,6 +369,18 @@ impl Error {
                     .body(body)
                     .unwrap()
             }
+            Self::Query(query_executor::Error::DatabaseNotFound { .. }) => {
+                let err: ErrorMessage<()> = ErrorMessage {
+                    error: self.to_string(),
+                    data: None,
+                };
+                let serialized = serde_json::to_string(&err).unwrap();
+                let body = Body::from(serialized);
+                Response::builder()
+                    .status(StatusCode::NOT_FOUND)
```

Comment:
Will it be easy enough to get a test added to assert the status?

Reply:
Yeah should be easy enough! Surprised we don't already have one now that I think about it